docs/DOCS_README.md

```diff
@@ -7,6 +7,13 @@ parent:
 
 If you want to open a PR on Gaia to update the documentation, please follow the guidelines in the [`CONTRIBUTING.md`](https://github.com/cosmos/gaia/tree/main/CONTRIBUTING.md)
 
+## Internationalization
+
+- Translations for documentation live in a `docs/<locale>/` folder, where `<locale>` is the language code for a specific language. For example, `zh` for Chinese, `ko` for Korean, `es` for Spanish, etc.
```

Comment:
`docs/<locale>/` ->  `docs/translations/<locale>/`

Reply:
This PR removes the `/translations/` subfolder as that actually doesn't allow translated text to render on built docs!

Reply:
ha ,ok!